src/umpire/resource/MemoryResourceRegistry.hpp

```diff
@@ -38,6 +40,8 @@ class MemoryResourceRegistry {
   MemoryResourceRegistry() noexcept;
 
   std::vector<std::unique_ptr<MemoryResourceFactory>> m_allocator_factories;
+
```

Comment:
Since you're stealing my function, at least take the documentation with it!! lol
```suggestion
/*!
   * \brief Get the names for existing Resources.
   *
   * The Resource Manager dynamically populates available memory resource types with
   * m_resource_names based on what's available. This is used for determining 
   * allocator accessibility.
   *
   * \return The private member, m_resource_names.
   */
```

Reply:
Oh whoopsies, well.. take out the stuff about the Resource Manager

Reply:
Let's keep this in your branch, as discussed.